src/Compilers/CSharp/Portable/Emitter/Model/PEAssemblyBuilder.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Emit
 {
-    internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssembly
+    internal abstract class PEAssemblyBuilderBase : PEModuleBuilder, Cci.IAssemblyReference
```

Comment:
OK. I may want to finesse the distinction a little later, but I'll talk to you offline about that.